package.json

```diff
@@ -15,7 +15,8 @@
 		"agentkeepalive": "^2.0.3",
 		"log": "1.4.*",
 		"stdio": "^0.2.3",
-		"testing": "*"
+		"testing": "*",
+		"websocket": "*"
```

Comment:
IME it is better to depend on a specific version of each library, with the caret operator `^` in front. So it doesn't crash on any updates.